nixos/modules/installer/cd-dvd/channel.nix

```
@@ -31,7 +31,11 @@ let
 in
 
 {
-  nix.registry.nixpkgs.flake.outPath = builtins.path { name = "source"; path = pkgs.path; };
+  nix.registry.nixpkgs.to = {
+    type = "path";
+    path = nixpkgs;
```

```suggestion
    path = builtins.path { name = "source"; path = nixpkgs; };
```
This still matters afaict.
This was done in a previous PR to

---

Your message ends abruptly. If there's a good reason to keep this, I'd like to know.

As far as I can tell, `cleanSource` outputs a path named `source` so this is unnecessary.

---

Sorry about that.

You are correct, I have now tested it.

It might be worth doing it explicitly both for documentation and defensive programming purposes, but I will leave that up to you.

---

Not convinced on the defensive programming side, but you are right that this needs documentation. Will add a comment